Code/server/GameServer.h

```diff
@@ -78,6 +78,14 @@ struct GameServer final : Server
     {
         return m_isPasswordProtected;
     }
+    void SetPublic(bool aIsPublic) noexcept
```

Comment:
There is no need for this. the value if a server is "public" is contained by bAnnounceServer convar. if you need it from another cpp file, you can extern Convar<bool> name it.

Reply:
Removed it, and the reference to it that was in ServerListService